src/status_im/setup/build_hooks/worklets.clj

```diff
@@ -0,0 +1,65 @@
+(ns status-im.setup.build-hooks.worklets
+  (:require [clojure.set])
+  (:import (java.io BufferedReader InputStreamReader PrintWriter)))
+
+(defn- update-js-output
+  [build-state code-processed]
+  (reduce-kv (fn [prev-build-state ns-key new-code]
+               (assoc-in prev-build-state [:output ns-key :js] new-code))
+             build-state
+             code-processed))
+
+(defn- get-workletized-code!
+  [code-seq store-atom]
+  (doall
+   (pmap (fn [[[_ filepath :as ns-key] js-code]]
+           (println "Workletizing:" filepath) ;; TODO: debug remove
+           (try
+             (let [command       "node workletize-code.js"
+                   process       (.exec (Runtime/getRuntime) command)
+                   output-stream (.getOutputStream process)
+                   writer        (PrintWriter. output-stream)
+                   reader        (BufferedReader. (InputStreamReader. (.getInputStream process)))]
```

Comment:
I'm spawning a new process to transform each file, I'd like to reuse them, but I'm not too experienced working with this. I'd highly appreciate assistance on this.